qa/sys/centos/bootstrap.sh

```diff
@@ -2,11 +2,3 @@
 
 yum update
 yum install -y java-1.8.0-openjdk-devel.x86_64
-
-##
-# Install logstash manually from a URL
-##
-BRANCH=${LOGSTASH_BRANCH:-'master'}
```

Comment:
we have to see how to do it, but end of the day is the same as if I download of have the rpm already available after bootstrap time. Let me explain better, only thing that is under discussion here is, 

* For you download + install should happen in rspec
* For me download should happen in bootstrap + install should happen in rspec.

Does this makes sense to you?